mllib/src/test/scala/org/apache/spark/ml/classification/OneVsRestSuite.scala

```diff
@@ -158,7 +158,7 @@ class OneVsRestSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
   }
 
   test("SPARK-21306: OneVsRest should support setWeightCol") {
-    val dataset2 = dataset.withColumn("weight", lit(1))
+    val dataset2 = dataset.withColumn("weight", lit(1.0))
```

Comment:
@facaiy It doesn't matter, since ```weightCol``` supports any numeric type and will cast it to double type under the hood.